packages/client/mutations/RemoveOrgUserMutation.ts

```diff
@@ -113,61 +129,71 @@ export const removeOrgUserTeamUpdater: SharedUpdater<RemoveOrgUserMutation_team>
   payload,
   {atmosphere, store}
 ) => {
-  const removedUserId = getInProxy(payload, 'user', 'id')
+  const removedUserId = payload.getLinkedRecord('user').getValue('id')
   const {viewerId} = atmosphere
   if (removedUserId === viewerId) {
     const teams = payload.getLinkedRecords('teams')
-    const teamIds = getInProxy(teams, 'id')
+    const teamIds = teams.map((team) => team.getValue('id'))
     handleRemoveTeams(teamIds, store)
   } else {
     const teamMembers = payload.getLinkedRecords('teamMembers')
-    const teamMemberIds = getInProxy(teamMembers, 'id')
+    const teamMemberIds = teamMembers.map((teamMember) => teamMember.getValue('id'))
     handleRemoveTeamMembers(teamMemberIds, store)
   }
 }
 
-export const removeOrgUserTaskUpdater = (payload, store, viewerId) => {
-  const removedUserId = getInProxy(payload, 'user', 'id')
+export const removeOrgUserTaskUpdater: SharedUpdater<RemoveOrgUserMutation_task> = (
+  payload,
+  {atmosphere, store}
+) => {
+  const removedUserId = payload.getLinkedRecord('user').getValue('id')
   const tasks = payload.getLinkedRecords('updatedTasks')
-  if (removedUserId === viewerId) {
-    const taskIds = getInProxy(tasks, 'id')
+  if (removedUserId === atmosphere.viewerId) {
+    const taskIds = tasks.map((task) => task.getValue('id'))
     handleRemoveTasks(taskIds, store)
   } else {
     handleUpsertTasks(tasks, store)
   }
 }
 
-export const removeOrgUserTeamOnNext = (payload, context) => {
+export const removeOrgUserTeamOnNext: OnNextHandler<RemoveOrgUserMutation_team> = (
+  payload,
+  context
+) => {
   const {atmosphere} = context
   const {teams} = payload
+  if (!teams) return
   teams.forEach((team) => {
-    const {newMeeting} = team
-    if (!newMeeting) return
```

Comment:
caught a bug! `newMeeting` doesn't even exist on the payload, so this bit of code was never reached